### DISCUSSION TITLE
How to enable extensions automatically

### DISCUSSION THREAD

**Discussion by chaaridev** [supabase/supabase#14170] [Questions] [answered]
Hi,
I'm using the self-hosted version Supabase using docker contained.
I want to ensure that all the extensions I need are automatically enabled when I start my docker containers.

Is it possible to do this with environment variables in the ".env" file or using a typescript that uses Supabase rest API to load my extensions?

I couldn't find this information in the documentation and the discussion and issues posted on GitHub.

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/self-hosting#extensions

**Reply by chaaridev** [reply]
Thank you @GaryAustin1  for the quick reply

**Reply by Wafje** [reply]
@GaryAustin1 has this documentation moved? Can you point me in the right direction?

**Reply by tvogel** [reply]
Same question here. There is no #extensions section there anymore.

**Reply by radinhamidi** [reply]
You gotta go back in time :) 
[https://web.archive.org/web/20230412124754/https://supabase.com/docs/guides/self-hosting#extensions](https://web.archive.org/web/20230412124754/https://supabase.com/docs/guides/self-hosting#extensions)

**Comment by homerjam** [comment]
Using SQL (which works with local/docker version of supabase):
```sql
 -- Example: enable the "vector" extension.
create extension vector
with
  schema extensions;

-- Example: disable the "vector" extension
drop
  extension if exists vector;
```
